### ISSUE TITLE
$or filter doesn't group together conditions

### ISSUE THREAD

**Issue by jayalfredprufrock** [feathersjs/feathers-knex#71]
This is a bit of a security issue. (See feathersjs/feathers#404)

I was able to fix this by altering a few lines in the `knexify()` method to use knex's grouping capabilities. My fork is all screwed up right now so I can't submit a PR at the moment, but I wanted to go ahead and post the fix here:

````
// line 69
if (method) {
    if (key === '$or') {
        return query.where(function(){
            value.forEach(condition => this[method].call(this, condition));
        });
     }
     return query[method].call(query, column, value);
 }
````

**Comment by kaiquewdev** [comment]
https://github.com/feathersjs/feathers-knex/pull/73

**Comment by daffl** [comment]
This should be closed now thanks to @kaiquewdev #73 and #75
